public/app/features/live/live.ts

```diff
@@ -29,19 +29,21 @@ class CentrifugeSrv implements GrafanaLiveSrv {
     });
     this.centrifuge.connect(); // do connection
     this.connectionState = new BehaviorSubject<boolean>(this.centrifuge.isConnected());
+    this.connectionBlocker = new Promise<void>(resolve => {
```

Comment:
@kaydelaney -- I made this a property rather than create a new one each time, is that what you mean?